docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -109,47 +109,55 @@ A *sink* connector allows you to send data from {product-kafka} to an external s
+*** Your OpenShift Dedicated trial cluster expires after 60 days
+*** Requires a cluster administrator to install the Connectors addon, as described in https://access.redhat.com/documentation/en-us/openshift_connectors/1/guide/15a79de0-8827-4bf1-b445-8e3b3eef7b01[Adding the Red Hat OpenShift Connectors add-on to your OpenShift Dedicated trial cluster^].
 
-[role="_abstract"]
 ifdef::qs[]
-Before you can use {product-connectors}, you must complete the link:https://console.redhat.com/application-services/learning-resources?quickstart=getting-started[Getting started with {product-long-kafka}] quick start to set up the following components:
+* Complete the steps in link:https://console.redhat.com/application-services/learning-resources?quickstart=getting-started[Getting started with {product-long-kafka}] quick start to set up the following components:
 
-* A *Kafka instance* that you can use for {product-connectors}.
-* A *Kafka topic* to store messages sent by data sources and make the messages available to data sinks.
-* A *service account* that allows you to connect and authenticate your {connectors} instances with your Kafka instance.
-* *Access rules* for the service account that defines how your {connectors} instances can access and use the topics in your Kafka instance.
+** A *Kafka instance* that you can use for {product-connectors}.
+** A *Kafka topic* to store messages sent by data sources and make the messages available to data sinks.
+** A *service account* that allows you to connect and authenticate your {connectors} instances with your Kafka instance.
+** *Access rules* for the service account that defines how your {connectors} instances can access and use the topics in your Kafka instance.
 endif::[]
+
 ifndef::qs[]
-Before you can use {product-connectors}, you must complete the steps in _{base-url}{getting-started-url-kafka}[Getting started with {product-long-kafka}^]_ to set up the following components:
+* Complete the steps in _{base-url}{getting-started-url-kafka}[Getting started with {product-long-kafka}^]_ to set up the following components:
 
-* A *Kafka instance* that you can use for {product-connectors}.
-* A *Kafka topic* to store messages sent by data sources and make the messages available to data sinks.
-* A *service account* that allows you to connect and authenticate your {connectors} instances with your Kafka instance.
-* *Access rules* for the service account that defines how your {connectors} instances can access and use the topics in your Kafka instance.
+** A *Kafka instance* that you can use for {product-connectors}.
+** A *Kafka topic* to store messages sent by data sources and make the messages available to data sinks.
+** A *service account* that allows you to connect and authenticate your {connectors} instances with your Kafka instance.
+** *Access rules* for the service account that defines how your {connectors} instances can access and use the topics in your Kafka instance.
```

Comment:
Would it be better to conditionalise only the lead-in sentence that's different, so that you don't have to repeat the 4 sub-bullet points that are identical? I'm not very familiar with AsciiDoc, so maybe it's not possible to do it in that way, or maybe it's preferred to conditionalise at the larger level.

Reply:
yes - good suggestion